### DISCUSSION TITLE
Problem with accessory that use DHT11

### DISCUSSION THREAD

**Discussion by Vyeffe** [HomeSpan/HomeSpan#402] [Q&A]
I created a bridge accessory with 3 switch accessories and 2 sensor (temp and humidity) accessories.
When i flash the esp32 only with the switches every thing goes well, then when i flash with all the configuration, including the 2 sensors, flashing goes well but HomeKit can pair the device ("impossible to add accessory"), but the strange thing is that on the serial monitor i see "ACCESSORY PAIRED" and i see also the list of paired controller.

I attach also the configuration file of the 2 sensor.

```
#include <DHT.h>
#define DHTPIN 5
#define DHTTYPE DHT11

DHT dht(DHTPIN, DHTTYPE);

////////////////////////////////////

struct DEV_TEMP : Service::TemperatureSensor {              // TEMPERATURE SENSOR

  SpanCharacteristic *temp;

  DEV_TEMP() : Service::TemperatureSensor()
  {
    dht.begin();
    temp = new Characteristic::CurrentTemperature(10.0);
    temp->setRange(-50, 100);

  } // end constructor

  void loop() {

    if (temp->timeVal() > 5000) {
      float t = dht.readTemperature();
      temp->setVal(t);
    }
  } // update
};

////////////////////////////////////

struct DEV_HUM : Service::HumiditySensor {                // HUMIDITY SENSOR

  SpanCharacteristic *hum;

  DEV_HUM() : Service::HumiditySensor() {

    dht.begin();
    hum = new Characteristic::CurrentRelativeHumidity(50);
    hum->setRange(0, 100);

  } // end constructor

  void loop() {

    if (hum->timeVal() > 5000) {
      float h = dht.readHumidity();
      hum->setVal(h);
    }

  } // update
};
```

**Comment by HomeSpan** [comment]
If the Home App thinks the device is not paired but HomeSpan says it is, type 'H' into the Serial Monitor to clear the pairing info in HomeSpan (causes a reboot as well).
